modules/WES/Makefile

```diff
@@ -15,8 +15,9 @@ all: gen-schema test docs
 gen-schema:
 	mkdir -p $(BUILD_DIR)
 	mkdir -p $(DATAMODEL_DIR)
-	poetry run gen-project --config-file ../../config/config.yaml -d $(BUILD_DIR) $(SOURCE_SCHEMA_PATH)
-	mv $(BUILD_DIR)/*.py $(DATAMODEL_DIR)/
+	poetry run gen-python $(SOURCE_SCHEMA_PATH) > $(BUILD_DIR)/wes.py
+	cp $(BUILD_DIR)/wes.py $(DATAMODEL_DIR)/
+	poetry run gen-json-schema $(SOURCE_SCHEMA_PATH) > $(BUILD_DIR)/wes_schema.json
```

Comment:
Do we have a step-by-step guiide for adding new modules and where new lines/files are reqiored? This coiuld be in a sperate `CONTRIBUTING.md` file per https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors